src/lib.rs

```diff
@@ -32,6 +32,7 @@
 //! [c-api]: https://numpy.org/doc/stable/reference/c-api
 //! [ndarray]: https://numpy.org/doc/stable/reference/arrays.ndarray.html
 
+#![deny(missing_docs, missing_debug_implementations)]
```

Comment:
To make it easier for contributors to hack, should this be warn by default, and then deny in CI?

Reply:
These two I would like to keep as part of the source:

Personally, I usually silence `missing_docs` when adding new items by slapping a `/// TODO` onto them, but I do write the doc comments before handing it over to the CI.

Similarly, adding `#[derive(Debug)]` is usually simple enough and I think having early feedback is better than late feedback for missing things.

Finally, these are also a statement to readers of the code of the standard it aspires to. Similar to adding `#![forbid(unsafe_code)]` even though admittedly not as strong an aspiration.

Reply:
Ok 👍